Assets/Scripts/PlayerMovement.cs

```diff
@@ -7,6 +7,8 @@ public class PlayerMovement : MonoBehaviour
     private float moveInput; // detects left or right key when pressed
     private  Rigidbody2D ballRigidBody; // variable for rigid body (private to only access it from the script)
     private bool grounded; // ground variable changes from true or false when object enters or exits collision with the ground
+    public float maxJumps = 2; // variable for the maximum number of jumps allowed
+    public float remainingJumps = 2; // variable for the number of jumps remaning
```

Comment:
you can make `remainingJumps` private because you modify it directly in code. Also, you don't have to initialize public variables because you can do it directly from the editor so instead `public float maxJumps = 2;` you can write `public float maxJumps;` .